web/src/components/molecules/Schema/FieldModal/FieldDefaultProps/index.tsx

```diff
@@ -0,0 +1,88 @@
+import Upload from "antd/lib/upload/Upload";
+import { FormInstance } from "rc-field-form/lib/interface";
+import React from "react";
+
+import Form from "@reearth-cms/components/atoms/Form";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Input from "@reearth-cms/components/atoms/Input";
+import Select from "@reearth-cms/components/atoms/Select";
+import TextArea from "@reearth-cms/components/atoms/TextArea";
+
+import { FieldType } from "../../types";
+
+export interface Props {
+  selectedType: FieldType | null;
+  form: FormInstance<any>;
+}
+
+const FieldDefaultProps: React.FC<Props> = ({ selectedType, form }) => {
+  const { Option } = Select;
+  if (selectedType === "Text") {
+    return (
+      <>
```

Comment:
You don't need any of the fragments in these returns. Please remove them

Reply:
Also all of them are wrapped in the same Form.Item component and props. Please pull it out of the conditional statements

Reply:
@KaWaite , can we put it as a subtask for refactoring, I'll create a task for this.
It needs sometime to study re:earth code

Reply:
This isn't a re:earth code thing. Its just not how react fragments should be used.

Reply:
@KaWaite , done, but `Form.item` is inside the child's component because ant.d won't work otherwise.